cli/pcluster/config/mappings.py

```diff
@@ -655,40 +656,258 @@
     ])
 }
 
-CLUSTER = {
+CLUSTER_COMMON_PARAMS = [
+    # OrderedDict due to conditional defaults values
+    ("cluster_config_metadata", {
+        # NOTE: this is not exposed as a configuration parameter
+        "type": ClusterConfigMetadataCfnParam,
+        "cfn_param_mapping": "ClusterConfigMetadata",
+        "update_policy": UpdatePolicy.IGNORED,
+        "visibility": Visibility.PRIVATE,
+    }),
+    ("key_name", {
+        "cfn_param_mapping": "KeyName",
+        "validators": [ec2_key_pair_validator],
+        "update_policy": UpdatePolicy.UNSUPPORTED
+    }),
+    ("base_os", {
+        "type": BaseOSCfnParam,
+        "cfn_param_mapping": "BaseOS",
+        "allowed_values": ["alinux", "alinux2", "ubuntu1604", "ubuntu1804", "centos6", "centos7"],
+        "validators": [base_os_validator, architecture_os_validator],
+        "required": True,
+        "update_policy": UpdatePolicy.UNSUPPORTED
+    }),
+    ("scheduler", {
+        "cfn_param_mapping": "Scheduler",
+        "allowed_values": ["awsbatch", "sge", "slurm", "torque"],
+        "validators": [scheduler_validator],
+        "required": True,
+        "update_policy": UpdatePolicy.UNSUPPORTED
+    }),
+    # Master
+    ("master_instance_type", {
+        "default": "t2.micro",
+        "cfn_param_mapping": "MasterInstanceType",
+        "validators": [ec2_instance_type_validator],
+        "update_policy": UpdatePolicy.UNSUPPORTED,
+    }),
+    ("master_root_volume_size", {
+        "type": IntCfnParam,
+        "default": 25,
+        "allowed_values": ALLOWED_VALUES["greater_than_25"],
+        "cfn_param_mapping": "MasterRootVolumeSize",
+        "update_policy": UpdatePolicy(
+            UpdatePolicy.UNSUPPORTED,
+            fail_reason=UpdatePolicy.FAIL_REASONS["ebs_volume_resize"],
+            action_needed=UpdatePolicy.ACTIONS_NEEDED["ebs_volume_update"]
+        )
+    }),
+    # Compute fleet
+    ("compute_root_volume_size", {
+        "type": IntCfnParam,
+        "default": 25,
+        "allowed_values": ALLOWED_VALUES["greater_than_25"],
+        "cfn_param_mapping": "ComputeRootVolumeSize",
+        "update_policy": UpdatePolicy.COMPUTE_FLEET_STOP
+    }),
+    ("spot_bid_percentage", {
```

Comment:
this is for awsbatch only, shall we move to SIT?

Reply:
Done